deployment/stacks/parent.ts

```diff
@@ -68,63 +68,72 @@ export class ParentStack extends Stack {
       brokerString,
     });
 
-    const {
-      applicationEndpointUrl,
-      cloudfrontDistributionId,
-      distribution,
-      s3BucketName,
-      uiBucket,
-    } = createUiComponents({
-      deploymentConfigParameters,
-      ...commonProps,
-    });
-
-    const {
-      userPoolDomainName,
-      identityPoolId,
-      userPoolId,
-      userPoolClientId,
-    } = createUiAuthComponents({
-      ...commonProps,
-      oktaMetadataUrl,
-      applicationEndpointUrl,
-      restApiId,
-      bootstrapUsersPasswordArn,
-      customResourceRole,
-    });
+    if (process.env.CDK_DEFAULT_ACCOUNT !== "000000000000") {
```

Comment:
How would you feel about putting this logic into it's own file that looks like this:
```js
export function isLocalstack() {
  return process.env.CDK_DEFAULT_ACCOUNT == "000000000000"
}
```
That way we could import that and make it a tad more readable so it'd be like:

```js
if (!isLocalstack()) {
  //do something
}
```

Reply:
yeah, you said this. https://github.com/Enterprise-CMCS/macpro-mdct-seds/pull/15058#discussion_r1953246987

sounds good.